### PR TITLE
Temporary fix for using the share operation when using linked slicers

### PR DESCRIPTION
**Pull request by gl3nn**

### CODE REVIEW

File: fireant/dataset/filters.py [hunk 1]
```diff
@@ -88,7 +88,7 @@ def __init__(self, field, start, stop):
 
     @property
     def definition(self):
-        return self.field.definition[self.start : self.stop]
+        return self.field.definition[self.start: self.stop]
```
Comment (mikeengland) [inline fireant/dataset/filters.py]: Minor: Seems like strange formatting. Not sure Black is setup on Fireant yet.

File: fireant/queries/builder/dataset_blender_query_builder.py [hunk 2]
```diff
@@ -281,7 +283,18 @@ def sql(self):
 
         # If data blending is unnecessary, then behave just like a regular dataset
         if len(dataset_queries) == 1:
-            return dataset_queries[0].sql
+            dataset_query_builder = dataset_queries[0]
+
```
Comment (mikeengland) [inline fireant/queries/builder/dataset_blender_query_builder.py]: It'd be great if these two file changes could have some test cases!

File: fireant/queries/builder/dataset_query_builder.py [hunk 3]
```diff
@@ -87,6 +87,13 @@ def sql(self):
         operations = find_operations_for_widgets(self._widgets)
         share_dimensions = find_share_dimensions(self._dimensions, operations)
 
+        return self.make_query(
```
Comment (mikeengland) [inline fireant/queries/builder/dataset_query_builder.py]: Strange that no query was returned here before!

File: fireant/dataset/filters.py (hunk 1 above)
Comment (gl3nn) [reply fireant/dataset/filters.py]: My IDE did this. I guess no space either after the colon would make more sense.

**Pull request merged**